Project-II-Calculator/calculator/src/components/ButtonComponents/Button.css

```diff
@@ -1,125 +1,114 @@
+/*All Buttons*/
+
+.button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    border: 1px solid #9a9a9a;
+}
```

Comment:
I think you can put background color to `.button` too instead of giving the same background to the each button.